packages/client/wallets/smart-wallet/package.json

```diff
@@ -31,7 +31,7 @@
         "@crossmint/common-sdk-base": "0.0.13",
         "@datadog/browser-logs": "4.42.2",
         "@zerodev/ecdsa-validator": "5.3.1",
-        "@zerodev/passkey-validator": "5.3.5",
+        "@zerodev/passkey-validator": "5.4.0",
```

Comment:
I noticed that passkey signature validation was busted, so I looked through ZeroDevs github and noticed this [PR](https://github.com/zerodevapp/sdk/pull/171).